packages/services/src/competition.service.ts

```diff
@@ -899,11 +943,18 @@ export class CompetitionService {
     }
 
     // Map enriched entries back to LeaderboardEntry format for return
-    return enrichedEntries.map((entry) => ({
-      agentId: entry.agentId,
-      value: entry.score, // Convert score back to value for LeaderboardEntry
-      pnl: entry.pnl,
-    }));
+    return enrichedEntries.map((entry) => {
+      const value =
+        "totalEquity" in entry
```

Comment:
imo we shouldnt use `"..." in <var>` in our code. we should type the `enrichedEntries` array so we can make the logical check in the ternary

Reply:
also resolved